sr_description/hand/xacro/forearm/forearm.urdf.xacro

```diff
@@ -1,40 +1,93 @@
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro">
   <!-- muscle is true for muscle hand or false for motor hand -->
+  <!-- hand_type can be either hand_e (default) or hand_e_plus -->
   <!-- prefix can be anything but usually is "r_" or  "l_" for right and left hands distinction -->
-  <xacro:macro name="forearm" params="muscle prefix">
+  <xacro:macro name="dark_grey" >
+    <material name="dark_grey">
+      <color rgba="0.21961 0.21961 0.21961 1.0" />
+    </material>
+  </xacro:macro>
+  <xacro:macro name="light_grey" >
+    <material name="light_grey">
+      <color rgba="0.90000 0.90000 0.90000 1.0" />
+    </material>
+  </xacro:macro>
+
+  <xacro:macro name="forearm" params="muscle hand_type:='hand_e' prefix">
```

Comment:
Is it Ok here to have a parameter with a default vale that precedes a parameter without a default value?
This is normally not allowed in python or C++.

Reply:
Sorry, just realized the parameters are always passed by name, not position. Forget it.